### PR TITLE
Preview Deployments

### PR DESCRIPTION
**Pull request by benjdlambert**
#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: .tugboat/config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+services:
+  backstage:
+    image: tugboatqa/node:lts
+    commands:
+      init:
+        - yarn
```
Comment (freben) [inline .tugboat/config.yml:6]: +install

File: .tugboat/config.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+        - yarn build
```
Comment (freben) [inline .tugboat/config.yml:7]: is that needed? hm i thought it would be like yarn tsc here but i'm frankly not sure

**Review by freben — APPROVED** [review]

File: .tugboat/config.yml (hunk 2 above)
Comment (Rugvip) [reply .tugboat/config.yml:7]: Would be needed to serve the frontend, although there's no config set up for that to work really ^_>. `yarn tsc` does indeed need to be called first as well.

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-lw0pmvx9oiect2nruf40nqhmaiihi2mb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5fc80ebd52062d1465824246

**Comment by Rugvip** [comment]
@﻿b﻿е﻿n﻿ј﻿а﻿m﻿і﻿n﻿r is this something you're still working on?

**Comment by benjdlambert** [comment]
Hahah even you tagged the wrong person! Yes - was gonna pick it up when I have some time. Maybe I have some extra time this goalie week :)

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-lw0pmvx9oiect2nruf40nqhmaiihi2mb.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5fc80ebd52062d1465824246

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-obpflkwgpooppnifveiqcqp1uokppjif.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60107554483a2b49fcf96785

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-xgfar9qqk9qwfwiay7aqqymcq5tugyt7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011318c564a7773fdb3cc5e

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-xgfar9qqk9qwfwiay7aqqymcq5tugyt7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011318c564a7773fdb3cc5e

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-xgfar9qqk9qwfwiay7aqqymcq5tugyt7.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011318c564a7773fdb3cc5e

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-mnjdi5nxd0oin5jov37ebjtqicxgid1z.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/601164ffaa6cfb8696706579

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-hu04quturqha7y7jmjokhebqofojn3nm.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60116cdb6541716c3214e147

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-u9ypm6eaariassnpx8usf1nzjnxkgge2.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/60116f15654171e52714e1b9

**Comment by benjdlambert** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-rnebxws9ic4nbbmrklw5rn4gxwc7xaqe.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011709b654171ae8d14e722

**Review by Rugvip — APPROVED** [review]
Noice :shipit:

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-gn2lmcfzkyy4xdywat0t9w3auvaz2kdp.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011f194564a773314b4aff9

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-gn2lmcfzkyy4xdywat0t9w3auvaz2kdp.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011f194564a773314b4aff9

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-kjoe2dvj7ru6v7sn5zyoddiygmylzxlo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011fca8564a775887b4b6b8

**Comment by backstage-service** [comment]
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr3484-kjoe2dvj7ru6v7sn5zyoddiygmylzxlo.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/6011fca8564a775887b4b6b8

**Pull request merged**